File: csharp/Practice01/src-orig/MyDriver/MyData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace MyDriver
{
    public sealed class MyData
    {
        public MyData(int queryId, string value)
        {
            this.QueryID = queryId;
            this.Value = value;
        }

        public int QueryID { get; private set; }

        public string Value { get; private set; }

        public override string ToString()
        {
            return this.QueryID + ", " + this.Value;
        }
    }
}

File: java/Practice01-Begin/src/myDriver/MyData.java
package myDriver;

public final class MyData {
    
    public final int queryId;
    public final String value;
    
    public MyData(int queryId, String value) {
        this.queryId = queryId;
        this.value = value;
    }
    
    public String toString() {
        return this.queryId + ", " + this.value;
    }
}
